00:00:00.007303 main     VirtualBox COM Server 6.1.36 r152435 win.amd64 (Jul 19 2022 21:51:27) release log
00:00:00.007306 main     Log opened 2022-08-02T18:40:07.878997600Z
00:00:00.007308 main     Build Type: release
00:00:00.007314 main     OS Product: Windows 11
00:00:00.007316 main     OS Release: 10.0.22000
00:00:00.007317 main     OS Service Pack: 
00:00:00.061847 main     DMI Product Name: Inspiron 3501
00:00:00.082123 main     DMI Product Version: 
00:00:00.082219 main     Firmware type: UEFI
00:00:00.083465 main     Secure Boot: Enabled
00:00:00.083617 main     Host RAM: 7914MB (7.7GB) total, 1140MB available
00:00:00.083648 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.083649 main     Process ID: 13580
00:00:00.083650 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.322598          VirtualBox: object creation starts
00:00:00.323222          Home directory: 'C:\Users\Dell\.VirtualBox'
00:00:00.323655          Installed Drivers:
00:00:00.370698            C:\WINDOWS\system32\DRIVERS\VBoxSup.sys (Version: 6.1.36.2435)
00:00:00.371176            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.36.2435)
00:00:00.371602            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.36.2435)
00:00:00.372006            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.36.2435)
00:00:00.373304          Loading settings file "C:\Users\Dell\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.377982          usbLibGetDevices: Starting USB device enumeration
00:00:00.394646          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:00:00.394766          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:00.399030          usbLibGetDevices: Found 2 USB devices, 0 captured
00:00:00.399252          USB: Unknown USB device detected (idVendor: 0x0c45, idProduct: 0x671e)
00:00:00.399435          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0aaa)
00:00:00.532704          netIfIsWireless: CreateFile on '\\.\{2F54F43E-3696-498C-B582-D2DCFC9A4AF0}' failed with rcWin=2 (0x2) - ignoring
00:00:00.542818          HostDnsMonitor: initializing
00:00:00.551215          HostDnsMonitor: updating information
00:00:00.551237          HostDnsMonitor: old information
00:00:00.551238            no server entries
00:00:00.551239            no domain set
00:00:00.551239            no search string entries
00:00:00.551240          HostDnsMonitor: new information
00:00:00.551240            server 1: 10.0.0.80
00:00:00.551241            domain: phub.net.cable.rogers.com
00:00:00.551242            no search string entries
00:00:00.635178          VD: VDInit finished with VINF_SUCCESS
00:00:00.638173          Loading settings file "C:\Users\Dell\VirtualBox VMs\Fedora\Fedora.vbox" with version "1.16-windows"
00:00:00.641489          VirtualBox: object created
00:00:01.930061          Saving settings file "C:\Users\Dell\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:01.940353          Saving settings file "C:\Users\Dell\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:03.878348          Launched VM: 71087664 pid: 2848 (0xb20) frontend: GUI/Qt name: Fedora
00:00:17.146797          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:17.693193          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:00:17.942659          netIfIsWireless: CreateFile on '\\.\{2F54F43E-3696-498C-B582-D2DCFC9A4AF0}' failed with rcWin=2 (0x2) - ignoring
00:00:18.835649          netIfIsWireless: CreateFile on '\\.\{2F54F43E-3696-498C-B582-D2DCFC9A4AF0}' failed with rcWin=2 (0x2) - ignoring
01:08:57.235004 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:08:57.276282 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:08:57.280570 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:09:44.001526 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:09:44.021249 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
01:09:44.022189 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:09:44.023445 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:26:47.479104          Saving settings file "C:\Users\Dell\VirtualBox VMs\Fedora\Fedora.vbox" with version "1.16-windows"
01:30:18.184591 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:30:18.467564 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:30:18.469741 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:30:21.854415 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:30:21.877614 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:30:21.877833 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:30:26.877716 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:30:26.880976 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:30:26.881101 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:49:43.551910 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:49:43.555233 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:49:43.555561 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
05:25:22.317398 dns-monitor HostDnsMonitor: updating information
05:25:22.317472 dns-monitor HostDnsMonitor: unchanged
05:25:54.954757 USBPROXY usbLibGetDevices: Starting USB device enumeration
05:25:54.993203 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
05:25:54.993372 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
05:25:54.993841 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
05:26:24.953599 USBPROXY usbLibGetDevices: Starting USB device enumeration
05:26:24.970728 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
05:26:24.970885 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
05:26:24.971168 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
05:26:28.485802 USBPROXY usbLibGetDevices: Starting USB device enumeration
05:26:28.490899 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
05:26:28.491254 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
05:27:17.236942 USBPROXY usbLibGetDevices: Starting USB device enumeration
05:27:17.259103 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
05:27:17.259263 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
05:27:17.259545 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
07:00:34.277723 USBPROXY usbLibGetDevices: Starting USB device enumeration
07:00:34.281816 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
07:00:34.283086 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
07:01:19.419301 USBPROXY usbLibGetDevices: Starting USB device enumeration
07:01:19.446709 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
07:01:19.446860 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&c2333a7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
07:01:19.447127 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
07:40:01.505739          Saving settings file "C:\Users\Dell\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
07:40:01.510791          Saving settings file "C:\Users\Dell\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
07:40:07.052807 main     VirtualBox: object deletion starts
07:40:07.053351 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\Dell\VirtualBox VMs\Fedora\Fedora.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
07:40:07.058505 main     HostDnsMonitor: shutting down ...
07:40:07.058943 main     HostDnsMonitor: shut down
07:40:07.062413 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
07:40:07.062864 main     VirtualBox: object deleted
